openmedia/player/player.py

```diff
@@ -144,7 +179,30 @@ def add(self, track_path):
         return False
 
     def get_current_second(self):
-        return self.pipeline.query_position(3)[1] / 10**9
+        return self.pipeline.query_position(Gst.Format.TIME)[1] / 10**9
```

Comment:
I think we should use `Gst.SECOND` everywhere we used `10**9` just to make the code clearer.